README.md

```diff
@@ -1 +1,51 @@
-# angular-custom-validation-readme
\ No newline at end of file
+# Custom validation with $validators
+
+## Overview
+
+The built-in validation in HTML forms can only go so far. However, we can actually define custom validation rules!
+
+## Objectives
+
+- Describe the $validators pipeline
+- Write a custom validator
+- Integrate the custom validator with ngMessages
+
+## $validators
+
```

Comment:
Add some more context here about the limitations of built-in HTML validations. List off a few different things that you may need to validate that HTML alone can't do for you.